src/middlewared/middlewared/plugins/pool_/dataset_processes.py

```diff
@@ -41,9 +41,12 @@ async def processes(self, oid):
         dataset = await self.middleware.call('pool.dataset.get_instance_quick', oid, {'encryption': True})
         if dataset['locked']:
             return []
-        path = attachments_path(dataset)
-        zvol_path = f'/dev/zvol/{dataset["name"]}'
-        return await self.middleware.call('pool.dataset.processes_using_paths', [path, zvol_path])
+
+        paths = [f'/dev/zvol/{dataset["name"]}']
```

Comment:
Hmm.... is this a place where we missed conversion to using `zvol_name_to_path()` ? 

https://github.com/truenas/middleware/blob/master/src/middlewared/middlewared/plugins/zfs_/utils.py#L19

Reply:
Great catch! We actually had a few places like this